addons/delivery/views/delivery_view.xml

```diff
@@ -294,6 +294,9 @@
                   <xpath expr="//field[@name='product_uom_qty']" position="after">
                       <field name="product_qty" invisible="1"/>
                   </xpath>
+                  <xpath expr="//field[@name='picking_policy']" position="after">
+                      <field name="invoice_shipping_on_delivery"/>
```

Comment:
Nope, this field is purely technical, no need to display it => that's a functional change

`invoice_shipping_on_delivery` was computed totally automatically in v9-10-11, I would check with the product owners if we really want to allow people to edit that